apps/web/src/pages/Profile/ExperienceFormPage/ExperienceFormPage.tsx

```diff
@@ -467,7 +467,15 @@ export const ExperienceForm = ({
 };
 
 const context: Partial<OperationContext> = {
-  additionalTypenames: ["Skill", "SkillFamily"], // This lets urql know when to invalidate cache if request returns empty list. https://formidable.com/open-source/urql/docs/basics/document-caching/#document-cache-gotchas
+  additionalTypenames: [
+    "Skill",
+    "SkillFamily",
+    "AwardExperience",
+    "CommunityExperience",
+    "EducationExperience",
+    "PersonalExperience",
+    "WorkExperience",
+  ], // This lets urql know when to invalidate cache if request returns empty list. https://formidable.com/open-source/urql/docs/basics/document-caching/#document-cache-gotchas
   requestPolicy: "cache-first", // The list of skills will rarely change, so we override default request policy to avoid unnecessary cache updates.
```

Comment:
This comment is probably worth updating now that it is not just skills in `additionalTypenames`.

Reply:
Good suggestion :+1:  done in [update comment to be more accurate](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9938/commits/b534cf69be6529eb1cae6ae55a7d40a3bfd769e3)